src/js/background_scripts/message-listener-utils.js

```diff
@@ -66,4 +66,81 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             }
         });
     };
+
+    /**
+     *
+     * Open a new window.
+     *
+     * @param {Boolean} active - Whether the new window should be focused when created.
+     * @param {String} homepageURL - The URL for the new window.
+     *
+     */
+    gamepad.messageListenerUtils.openNewWindow = function (active, homepageURL) {
+        chrome.windows.getAll(function (windowsArray) {
```

Comment:
Please leave a TODO here indicating that this should be removed once #34 is addressed.